tests/rust/linestyle_crlf.toml

```diff
@@ -0,0 +1,2 @@
+language = "c"
```

Comment:
Any reason to force `language = "c"`? (that forces even the cpp to be c)

Reply:
Thanks; no! Will fix. The language here doesn't even matter; I was just trying to cut down on the amount of generated content. (Since all we care about is the line ending.)